src/OnePlusBot/Modules/Administration.cs

```diff
@@ -25,11 +25,37 @@ public class Administration : ModuleBase<SocketCommandContext>
         ]
         public async Task<RuntimeResult> OBanAsync(ulong name, [Remainder] string reason = null)
         {
-            var modlog = Context.Guild.GetTextChannel(Global.Channels["modlog"]);
             await Context.Guild.AddBanAsync(name, 0, reason);
 
             MuteTimerManager.UnMuteUserCompletely(name);
 
+              
+            if(reason == null)
+            {
+                reason = "No reason provided.";
```

Comment:
Don't waste space like that, keep it indented, it's no big deal but generally preferred.